proto/redpanda/api/dataplane/v1alpha2/pipeline.proto

```diff
@@ -37,11 +40,10 @@ message Pipeline {
     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "The configuration of the Pipeline as YAML. See https://docs.redpanda.com/redpanda-connect/configuration/about for more details."}
   ];
 
-  // The minimum amount of resources needed for the pipeline to run
-  Resources request = 5 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "The minimum amount of resources needed for the pipeline to run."}];
-
-  // The maximum amount of resources that should be assigned to the pipeline
-  Resources limit = 6 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "The maximum amount of resources that should be assigned to the pipeline."}];
+  // The amount of resources that is requested for the pipeline. Depending on
+  // the backend, the resources might be guaranteed, or might be tweaked based
```

Comment:
nit: it's always "guaranteed", just in a "different way".. we may have to shuffle some pods a bit to guarantee it, but the eventual target is the same. It's a bin packing problem we need to solve

Reply:
Comment adjusted